.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+   <!-- If you need help with you PR or if you want to to discuss something with us, join the WalkingPal [Discord community](https://discord.gg/Et3aQTsW6a)! -->
```

Comment:
See this file carefully 
https://github.com/WalkingPal/walking-pal-web/pull/77/files#diff-a3ca026a51f9e3a25d135e0d982c895b4184928ae9f6f2c001a24c58b0f688ca

There are just 6 lines. you should just put these 6 line in you template. Also, if you want to add something to these 6 lines "as comments", please do add